python/calico/felix/endpoint.py

```diff
@@ -887,16 +893,26 @@ def _apply_endpoint_update(self):
 
             tiers = pending_endpoint.get("tiers", [])
             tier_dict = OrderedDict()
+            raw_tier_dict = OrderedDict()
             for tier in tiers or []:
                 pols = []
+                raw_pols = []
                 for pol in tier["policies"] or []:
                     pol_id = TieredPolicyId(tier["name"], pol)
-                    pols.append(pol_id)
+                    if is_untracked(pol_id):
+                        raw_pols.append(pol_id)
+                    else:
+                        pols.append(pol_id)
                 tier_dict[tier["name"]] = pols
+                raw_tier_dict[tier["name"]] = raw_pols
```

Comment:
Ah no, I believe I've avoided that problem, already, by adding a 'default_drop=True' parameter to host_endpoint_updates in fiptgenerator.py, and plumbing that through from LocalEndpoint:_update_chains (https://github.com/neiljerram/felix/blob/untracked/python/calico/felix/endpoint.py#L948).

Do you agree, or do you still see a hole here?